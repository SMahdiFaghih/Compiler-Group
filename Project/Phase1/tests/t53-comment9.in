some code // and some comment /* in whatever form you please
and then some more code