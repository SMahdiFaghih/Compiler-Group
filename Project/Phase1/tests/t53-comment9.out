T_ID some
T_ID code
T_ID and
T_ID then
T_ID some
T_ID more
T_ID code
